package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "soundclout-active-player-module",
+  "version": "1.0.0",
+  "description": "This features an animated player synced proportionally with a time-stamped location in a song. Its clickable components include a play/pause toggle button, fastforward/rewind buttons, shuffle/repeat buttons, hover volume control, album thumbnails, a like button, and a \"up next\" pop up menu.  The \"up next\" pop up menu also features a button to clear the list, a toggle to autoplay upcoming tracks, as well as like buttons and a pop-up options menu for each track.  The menu allows you to like, share, add or remove from \"next up\", add to a playlist, or start a radio station of related tracks.",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "db": "mysql -u root -pyourpassword < ./database/data/schema.sql",
+    "build": "webpack -w",
```

Comment:
Alternate implementation for this (depending on your version of webpack) would be to include an additional property within your webpack.config file. Feel free to look into it for future projects:
[offical webpack docs reference](https://webpack.js.org/configuration/watch/)